stl/inc/xmemory

```diff
@@ -1956,29 +1952,52 @@ struct _Wrap {
 };
 #pragma warning(pop)
 
-// STRUCT TEMPLATE _Alloc_temporary
+// STRUCT TEMPLATE _Alloc_temporary2
 template <class _Alloc>
-struct _Alloc_temporary {
+struct _Alloc_temporary2 {
     using value_type = typename _Alloc::value_type;
     using _Traits    = allocator_traits<_Alloc>;
 
     _Alloc& _Al;
+
+#ifdef __cplusplus_winrt
     union {
         _Wrap<value_type> _Storage;
     };
 
+    _NODISCARD _CONSTEXPR20 value_type& _Get_value() noexcept {
+        return _Storage._Value;
+    }
+
+    _NODISCARD _CONSTEXPR20 const value_type& _Get_value() const noexcept {
+        return _Storage._Value;
+    }
+#else // ^^^ workaround for "T^ is not allowed in a union" ^^^ / vvv no workaround vvv

```

Comment:
This is VSO-586813 if we'd like a number to refer to.

Reply:
I'll add these citations in a followup PR.